src/Microsoft.OData.Edm/ExtensionMethods/ExtensionMethods.cs

```diff
@@ -1597,6 +1597,95 @@ public static IEnumerable<IEdmStructuralProperty> Key(this IEdmEntityType type)
         }
 
         /// <summary>
+        /// Gets the declared alternate keys of the most defined entity with a declared key present.

```

Comment:
Move these extension methods to the server library: OData Web API, or create a new set of API.  Thanks.